string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,9 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    if (word[i].toLowerCase() !== word[i]) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
We can actually accomplish this without a loop! We can convert the whole word to lowercase and compare it to the original. Here is our snippet
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```